src/controllers/boardsController.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../models/index');
+const { Op } = require("sequelize");
+const User = require("../models/user")
+const Board = require("../models/board")
+
+module.exports = {
+    index: (req, res, next) => {
+        db.Board.findAll({
+            order: [
+                ['createdAt', 'DESC']
+            ],
+            include: [{
+                model: db.User,
+                required: true
+            }]
+        }).then(board => {
+            const data = {
+                title: 'Boards',
+                login:req.session.user,
+                content: board,
+            }
+            res.render('layout', { layout_name: 'boards/index', data});
+        });
+    },
+    add: (req, res, next) => {
+        const data = {
+            title: 'Boards/Add',
+            login:req.session.user,
+            err: null
+        }
+        res.render('layout', { layout_name: 'boards/add', data});
+    },
+    create: (req, res, next) => {
+        const form = {
+            userId: req.session.user.id,
+            title: req.body.title,
+            message:req.body.msg
+        };
+        db.sequelize.sync()
+        .then(() => db.Board.create(form)
+        .then(brd=>{
+            res.redirect('/boards');
+        })
+        .catch((err)=>{
+            const data = {
+                title: 'Boards',
+                login: req.session.user,
+                err: err,
+            }
+            res.render('layout', { layout_name: 'boards/add', data});
+        })
+        )
+    },
+    edit: async (req, res, next) => {
+        const BoardId = req.params.id;
+        await db.Board.findOne({
+          where:{
+            id: BoardId,
+          }
+        }).then(board => {
+          const data = {
+            title: 'Boards/Edit',
+            login:req.session.user,
+            board: board,
+            err: null
+          }
+          res.render('layout', { layout_name: 'boards/edit', data});
+        });      
+    },
+    update: async (req, res, next) => {
+        const BoardId = req.params.id;
+        await db.Board.update({
+                title: req.body.title,
+                message: req.body.msg
+        },
+        {
+            where: { id: BoardId, }
+        }).then(() => {
+            res.redirect('/boards');
+        }
+        ).catch((err)=>{
```

Comment:
```suggestion
        ).catch((err) => {
```
スペースを空けるべき箇所で空いていない部分がありますので、エディタの自動補正を使用すると良いです。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840

業務ではこのような箇所も良く見られます。
全体的に良く記述できているので、このような部分でマイナスな印象を与えてしまうのはもったいないです。

Reply:
ありがとうございます。formatter機能を使って自動補正を行いました。